core/src/test/java/nl/nn/adapterframework/pipes/JsonWellFormedCheckerTest.java

```diff
@@ -41,12 +33,19 @@ public JsonWellFormedChecker createPipe() {
 		return new JsonWellFormedChecker();
 	}
 
-	@Test
-	public void runTest() throws Exception {
+	@MethodSource("data")
+	@ParameterizedTest(name = "{index}: input [ {0} ] forward [{1}]")
+	public void runTest(String input, String forward) throws Exception {
+		initJsonWellFormedCheckerTest(input, forward);
 		pipe.registerForward( new PipeForward("failure", "path") );
 		pipe.configure();
 		pipe.start();
 		PipeRunResult pipeRunResult = doPipe(pipe, input, session);
 		assertEquals(forward, pipeRunResult.getPipeForward().getName());
 	}
+
+	public void initJsonWellFormedCheckerTest(String input, String forward) {
```

Comment:
Ik zou deze methodenaam niet op 'Test' laten eindigen, aangezien het geen test is. Zonder 'Test' of met 'TestData' zou dan beter zijn.